ocp_resources/datavolume.py

```diff
@@ -217,12 +217,15 @@ def wait_deleted(self, timeout=TIMEOUT_4MINUTES):
         super().wait_deleted(timeout=timeout)
         return self.pvc.wait_deleted(timeout=timeout)
 
-    def wait(self, timeout=TIMEOUT_10MINUTES, failure_timeout=TIMEOUT_2MINUTES):
-        self._check_none_pending_status(failure_timeout=failure_timeout)
+    def wait(self, wait_for_exists_only=False, timeout=TIMEOUT_10MINUTES, failure_timeout=TIMEOUT_2MINUTES):
```

Comment:
if someone is using positional args your PR breaks their code; add last

Reply:
You're right, thanks